worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -205,7 +207,12 @@ public void smoothBrush(Player player, LocalSession session,
         tool.setSize(radius);
         tool.setBrush(new SmoothBrush(iterations, mask), "worldedit.brush.smooth");
 
-        player.print(String.format("Smooth brush equipped (%.0f x %dx, using %s).", radius, iterations, mask == null ? "any block" : "filter"));
+        player.printInfo(TranslatableComponent.of(
+                "worldedit.brush.smooth.equip",
+                TextComponent.of((int) radius),
+                TextComponent.of(iterations),
+                TextComponent.of(mask == null ? "any block" : "filter")
```

Comment:
Shouldn't these be translated too? Either as two separate top-level keys (`.equip.anyblock`/`.equip.filter`) or passed as translated args (`worldedit.mask.none`/`worldedit.mask.applied`).